sky/utils/controller_utils.py

```diff
@@ -85,17 +88,20 @@ class Controllers(enum.Enum):
             f'sky spot cancel -a{colorama.Style.RESET_ALL}\n'),
         check_cluster_name_hint=(
             f'Cluster {spot_utils.SPOT_CONTROLLER_NAME} is reserved for '
-            'managed spot controller. '),
-        default_hint_if_non_existent='No managed spot jobs are found.')
+            'managed spot controller.'),
+        default_hint_if_non_existent='No in-progress spot jobs.',
+        hint_for_connection_error=(
+            'Failed to connect to spot controller, please try again later.'))
     SKY_SERVE_CONTROLLER = _ControllerSpec(
-        name='sky serve controller',
+        name='serve controller',
+        managing_name='services',
```

Comment:
I found this field pretty weird. Feels "shallow"? Used outside of this module but not e.g., L122. Possible to avoid it?

Reply:
Ahh, sorry. This is a remnant for a previous attempt for a refactorization. Removed it now. Thanks!